jvm/src/main/buf.gen.yaml

```diff
@@ -0,0 +1,26 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+version: v1
+plugins:
+   # Building the Python build and building the mypy interfaces.
+  - plugin: buf.build/protocolbuffers/python:v28.3
+    out: ../../../python/src/spark_rapids_ml/proto
+  - plugin: buf.build/grpc/python:v1.67.0
```

Comment:
Do these versions (v28.3 and v1.67.0) need to match exactly the java side in the pom.xml file ?